### PR TITLE
[Translation] Improved and completed the Arabic translation for the Website.

### PR DESCRIPTION
**Pull request by Ali-x98**
I have completed the Arabic translation for the website.

Please consider using an online platform such as Weblate to help make the translation process more accessible.

### CODE REVIEW

**Comment by Yokozuna59** [comment]
I'm not a reviewer. However, I'd like to point out that it is a fix for issue #1560 for Arabic.

**Review by Yokozuna59 — COMMENTED** [review]
As I said before, I'm neither a reviewer nor a contributor. But I've reviewed your changes. I'm suggesting these changes, and it's up to you to modify your PR according to them or not.

_**NOTE**_: I haven't reviewed `labelling.json` or `tasks.json` files.

File: website/public/locales/ar/common.json [hunk 1]
```diff
@@ -1,36 +1,52 @@
...
+  "status_dashboard": "لوحة الحالة",
+  "submit": "قدِّم",
+  "submit_your_answer": "قدِّم جوابك",
   "success": "نجاح",
   "terms_of_service": "شروط الخدمة",
-  "title": "Open Assistant (المساعد مفتوح المصدر)",
-  "user_leaderboard": "جدول المتصدرين من المستخدمين",
-  "users_dashboard": "لوحة عرض المستخدمين",
-  "users": "المستخدمين",
+  "title": "المساعد الحرُّ (Open Assistant)",
```
[the remark this answers is not in the document]
Comment (Ali-x98) [reply website/public/locales/ar/common.json:46]: I understand what Richard Stallman is saying. However, `المساعد مفتوح المصدر` is too verbose and heavy on the tongue.

I have also considered `المساعد المفتوح`. But using the word `مفتوح` can be inappropriate when referring to a job that can be traditionally achieved by a person. In this case, `Assistant` meaning `مساعد` .

Besides, I believe this project does campaign for the "freedom for the users of computing" by offering an alternative to the proprietary ChatGPT.

As such, I still consider `المساعد الحر` the better translation. (`المساعد الحر` translates to `The Free Assistant` or `Libre Assistant`)

That being said, I am thankful to you for taking the time to write this suggestion. If you can think of a better translation, then feel free to share it.

File: website/public/locales/ar/leaderboard.json [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
+  "total": "الإجماليُّ",
   "user": "المستخدم",
-  "view_all": "عرض الكل",
-  "weekly": "أسبوعياً"
+  "username": "اسم المستخدم",
+  "view_all": "اعرض الكلَّ",
+  "week": "الأسبوع",
+  "weekly": "أسبوعيًّا",
+  "your_account": "حسابك",
+  "your_stats": "إحصاءاتك"
```
[the remark this answers is not in the document]
Comment (Ali-x98) [reply website/public/locales/ar/leaderboard.json:35]: Please explain why you do not think it is grammatically correct.

File: website/public/locales/ar/message.json [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
-  "report_title": "تبليغ",
-  "send_report": "إرسال",
-  "stop_tree": "اوقف الشجرة",
-  "submit_labels": "إرسال",
-  "tree_stopped": "تم ايقاف الشجرة {{id}}",
-  "view_user": "عرض المستخدم",
-  "your_recent_messages": "رسائلك الاخيرة"
+  "recent_messages": "أحدث الرسائل في {{language}}",
+  "report_action": "بلِّغ",
+  "report_placeholder": "لماذا تجب مراجعة هذه الرسالة؟",
```
[the remark this answers is not in the document]
Comment (Ali-x98) [reply website/public/locales/ar/message.json:17]: Please explain why you believe that `تجب` is incorrect.

File: website/public/locales/ar/tos.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
-  "title": "شروط الخدمة ل Open Assistant (المساعد مفتوح المصدر)",
-  "content": "للاستمرار في استخدام Open Assistant (المساعد مفتوح المصدر)، يجب عليك قبول شروط الخدمة الخاصة بنا أولاً.",
-  "accept": "قبول",
-  "decline": "رفض"
+  "title": "شروط خدمة المساعد الحرِّ",
+  "content": "عليك قبول شروط خدمتنا لتستمرَّ في استخدام المساعد الحرِّ",
```
[the remark this answers is not in the document]
Comment (Ali-x98) [reply website/public/locales/ar/tos.json:3]: This is a good point. See my reply to the Stallman suggestion.

[the remark this answers is not in the document]
Comment (Ali-x98) [reply website/public/locales/ar/tos.json:3]:
```suggestion
  "title": "شروط خدمة المساعد الحرِّ",
  "content": "عليك قبول شروط خدمتنا لتستمرَّ في استخدام المساعد الحرِّ (Open Assistant)",
```

File: website/public/locales/ar/index.json [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
+    "q0": "كيف يسري تطوُّر المشروع حتَّى الآن؟",
+    "a0": "نحن في أوائل مراحل تطويره، ونعمل وفق أبحاث ركيزة في تطبيق التعليم المحفَّز من ملاحظات البشر (RLHF) على نماذج اللغات العملاقة.",
+    "q1": "من يعمل على تطوير المساعد الحرِّ؟",
+    "a1": "تنسِّق لايون (LAION) والمساهمون المشروع، وذلك رغبةً منهم في توفير هذه التقنية للجميع.",
+    "q2": "ما رخصة المشروع؟",
+    "a2": "يرخَّص المشروع وفق رخصة أباتشي 2.0 (Apache 2.0).",
+    "q3": "أستُنشر بيانات التدريب؟",
+    "a3": "نعم، بموجب ترخيص المشاع الإبداعيِّ (CC BY 4.0.)",
+    "q4": "هل سيكون المساعد الحرُّ مجانيًا؟",
+    "a4": "نعم، سيكون المساعد المفتوح مجانيًّا للاستخدام والتعديل.",
```
[the remark this answers is not in the document]
Comment (Ali-x98) [reply website/public/locales/ar/index.json:15]: See my reply about the naming. Nice catch tho.
```suggestion
    "a4": "نعم، سيكون المساعد الحرُّ مجانيًّا للاستخدام والتعديل.",
```

File: website/public/locales/ar/index.json [hunk 6]
```diff
@@ -1,23 +1,23 @@
...
-    "a2": "تم ترخيص الكود والموديلات تحت ترخيص Apache 2.0",
-    "q3": "هل سيتم نشر بيانات التدريب أيضًا؟",
-    "a3": "نعم، بموجب ترخيص CC BY 4.0.",
-    "q4": "هل سيكون المساعد المفتوح مجانيًا؟",
-    "a4": "نعم، سيكون المساعد المفتوح مجانيً للاستخدام والتعديل.",
-    "q5": "ما هو العتاد المطلوب لتشغيل الموديلات؟",
-    "a5": "سيكون هناك إصدارات يمكن تشغيلها على الأجهزة المصممة للمستهلكين."
+    "q0": "كيف يسري تطوُّر المشروع حتَّى الآن؟",
+    "a0": "نحن في أوائل مراحل تطويره، ونعمل وفق أبحاث ركيزة في تطبيق التعليم المحفَّز من ملاحظات البشر (RLHF) على نماذج اللغات العملاقة.",
+    "q1": "من يعمل على تطوير المساعد الحرِّ؟",
```
[the remark this answers is not in the document]
Comment (Ali-x98) [reply website/public/locales/ar/index.json:8]: This actually made me notice something else.
```suggestion
    "q1": "من يطوِّر المساعد الحرَّ؟",
```

File: website/public/locales/ar/dashboard.json [hunk 7]
```diff
@@ -1,8 +1,14 @@
...
   "dashboard": "لوحة المعلومات",
-  "evaluate": "تقييم",
-  "go": "ذهاب الى",
-  "grab_a_task": "ابدأ مهمة!",
-  "label": "وسم"
+  "go": "إذهب",
+  "welcome_message": {
+    "label": "أهلًا يا {{username}}! ",
+    "contributor": "مساهم",
+    "description": "المساعد الحرُّ هو برنامج ذكاء اصطناعيٍّ حرُّ المصدر يستخدم ويمرِّن نماذج لغات متقدِّمة ليفهم ويحادث البشر",
```
[the remark this answers is not in the document]
Comment (Ali-x98) [reply website/public/locales/ar/dashboard.json:11]:
```suggestion
    "description": "المساعد الحرُّ هو برنامج ذكاء اصطناعيٍّ مفتوح المصدر يستخدم ويمرِّن نماذج لغات متقدِّمة ليفهم ويحادث البشر",
```

**Comment by Ali-x98** [comment]
Thanks to @Yokozuna59 for taking the time to review the translation.

**Review by AbdBarho — APPROVED** [review]
Thanks @Ali-x98 @Yokozuna59.

I will merge this now, in case there are any changes or discussion points, you can still create a new PR and we can continue there.

**Pull request merged**